code/datums/uplink_items/uplink_nuclear.dm

```diff
@@ -498,10 +498,13 @@
 	uplinktypes = list(UPLINK_TYPE_NUCLEAR, UPLINK_TYPE_SST)
 
 /datum/uplink_item/suits/hardsuit/shielded
```

Comment:
This is no longer a hardsuit, update the path.

Reply:
updated